alert-policies/temporal-cloud/FailedWorkflows.yml

```diff
@@ -0,0 +1,30 @@
+# Name of the alert
+name: Failed Workflows
+
+# Description and details
+description: |+
+  This alert is triggered if the Temporal cloud workflows failes once within a 5 minutes window.
```

Comment:
```suggestion
  This alert is triggered if the Temporal cloud workflows failed once within a 5 minutes window.
```

Reply:
Hi @sjyothi54 ,
I have updated the review changes.